script/main.py

```diff
@@ -212,6 +212,37 @@ def percentage(conn, gender):
     print('Percentage of women: ' , percentage_of_women, '%')
   else:
     print('You are entering the wrong gender')
+
+
+def average_age(conn, gender):
```

Comment:
In general calculating the age is always better then getting age from the database. So this is better approach.